salt/pillar/vault.py

```diff
@@ -68,11 +88,12 @@ def __virtual__():
 
 def ext_pillar(minion_id,  # pylint: disable=W0613
                pillar,  # pylint: disable=W0613
-               conf):
```

Comment:
Hello, 

I went ahead and switched it back to "conf"  

It shouldn't have broken anyone's confs unless they were using an explicit call to the "conf" tag in their files.  The default structure for the external pillar config in the documentation for this wasn't set up for multiple arguments to the vault external pillar.  So, the documentation just followed the following format:  
- vault: path=secret/salt
no mention of the arguments.  Keeping the master configs in this same format wouldn't change anything and things would continue to function.  It would only be a problem if someone had changed their master configs to look like: 
- vault: 
    conf: path=secret/salt

However, you are right in that it would really be annoying for someone if they had gone through followed that secondary format (for consistency of confs or whatever other reason) and have it break here.  
It was meant as a nicety for clarity, but you are right that it isn't likely worth the effort.